### ISSUE TITLE
Keyboard user is unable to access the “Font color“ Sliders present under “text” pane.

### ISSUE THREAD

**Issue by sagar17sep** [OfficeDev/office-ui-fabric-react#11031]
### Environment Information

- **Package version(s)**: 7.24.0
- **OS version:** 1903 (OS Build: 18326.295)
- **Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If controls are not accessible with keyboard than keyboard user will not be able to access the control and will face difficulties while interacting with it.

**Repro steps:**
1. Open URL "https://make.test.powerapps.com" and login with valid credentials.
2. PowerApps Homepage will appear.
3. Navigate to template present at the bottom, then navigate to any already created portal app.
4. Navigate to context menu of portal app and select it. Select Edit (preview) from the context menu, portal will be opened on the screen.
5. Navigate to right pane 'Components' and select it.
6. Navigate to 'Text' under portal components on 'Components' pane and select it.
7. Changes will appear on right pane, navigate to right pane and try to edit Alignment, font color' of the text.
8. Navigate to “font color” sliders present under “Text” pane.
9. verify keyboard user is able to access the “Font color “ Sliders present under “text” pane or not.

**Actual behavior:**
Keyboard user is unable to access the “Font color “ Sliders present under “text” pane.

**Expected behavior:**
Keyboard user should able to access the “Font color “ Sliders present under “text” pane.

**Please find the attachment**
[MAS2.1.1-text font color sliders.wmv.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3792634/MAS2.1.1-text.font.color.sliders.wmv.zip)

Please reach out to Portal FX team for more details.

**Comment by maxwellred** [comment]
Hi @sagar17sep , This issue involves a product, PowerApps, our team neither builds nor maintains. For issues related to our product you are welcome to submit an official issue [here](https://github.com/OfficeDev/office-ui-fabric-react/issues/new/choose) and supplying a codepen demo of the respective control found [here](https://developer.microsoft.com/en-us/fabric#/controls/web) accompanied with the necessary repro steps. This issue will be labelled as "Resolution: External" and we would appreciate your help by closing this issue yourself. Thank you very much for your interest and support in the Office UI Fabric React project!
